ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -576,6 +576,29 @@
     "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"FULL_DEFENCE\"",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
Can we not make it a complextype screen instead of splitting the contents and thus introducing two fields in the backend civil-service CaseData for a single screen? If we can manage this in a single complex type like in some other screens, we can accommodate it in one single FutureApplicationIntention datatype in CaseData.

Reply:
Discussed and we agreed to keep the current implementation.